Dockerfile.tmpl

```diff
@@ -8,6 +8,9 @@ ARG TORCHAUDIO_VERSION
 ARG TORCHTEXT_VERSION
 ARG TORCHVISION_VERSION
 
+# Ensures shared libraries installed with conda can be found by the dynamic link loader.
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/opt/conda/lib"
```

Comment:
This line happens too early (before the base image FROM tag) which causes a docker build error.